core/src/main/scala/org/apache/spark/deploy/SparkHadoopUtil.scala

```diff
@@ -79,8 +79,10 @@ private[spark] class SparkHadoopUtil extends Logging {
    * Appends S3-specific, spark.hadoop.*, and spark.buffer.size configurations to a Hadoop
```

Comment:
nit. This function description seems to become behind, but, PR looks good.
Thanks!